app/test/client-config-related-entities-test.html

```diff
@@ -63,48 +62,50 @@
       assert.isString(dataout.phone.data[0]._id);
       assert.equal(dataout.phone.data[0]._type, 'phone');
       assert.equal(dataout.phone.data[0].title, '1234567890');
-      assert.equal(dataout.phone.data[0].subtitle, '2 offer(s)');
+      assert.equal(dataout.phone.data[0].subtitle, '');
     });
 
     test('defines email array with information specified in data-model', function() {
       assert.equal(dataout.email.count, 1);
       assert.isString(dataout.email.data[0]._id);
       assert.equal(dataout.email.data[0]._type, 'email');
       assert.equal(dataout.email.data[0].title, 'abc@xyz.com');
-      assert.equal(dataout.email.data[0].subtitle, '1 offer(s)');
+      assert.equal(dataout.email.data[0].subtitle, '');
     });
 
-    test('defines website array with information specified in data-model', function() {
-      assert.equal(dataout.webpage.count, 1);
-      assert.isString(dataout.webpage.data[0]._id);
-      assert.equal(dataout.webpage.data[0]._type, 'webpage');
-      assert.isString(dataout.webpage.data[0].title);
-      assert.isString(dataout.webpage.data[0].subtitle);
-      assert.isObject(dataout.webpage.data[0].details);
-      assert.isString(dataout.webpage.data[0].details.body);
-      assert.isString(dataout.webpage.data[0].details.url);
-      assert.isArray(dataout.webpage.data[0].details.addresses);
-      assert.equal(dataout.webpage.data[0].details.addresses[0], 'Los Angeles');
-      assert.equal(dataout.webpage.data[0].details.date, '2014-10-29T03:05:00');
-    });
+    // TODO REMOVE THIS AS THERE ARE NO WEBPAGE ENTITY PAGES ANYMORE
```

Comment:
If there are no longer webpage entities, please remove the webpage files, including app/webpage.html, app/test/client-config-webpage-test.html, and app/scripts/webpage-transform.js.  Additionally, please update server/config/elastic-types.js to say "offer" instead of "webpage."